### PR TITLE
[DOCS] Rephrase `watcher_admin` role desc

### LINKED ISSUE #68869: [DOCS] Rephrase built-in watcher_admin role's description
> `watcher_admin`
> Grants read access to the `.watches` index, read access to the watch history and the triggered watches index and allows to execute all watcher actions.

I was sure that this role acted as the admin (it does), but it starts by talking about `read` privileges. Can we swap the sentence around so that the important bit is at the front? Something like:

> Allows users to create and execute all watcher actions. Grants `read` access to the `.watches` index, read access to the watch history and the triggered watches index.

Relates to https://github.com/elastic/stack-docs/pull/190

### PR DESCRIPTION
**Pull request by jrodewig**
Closes #68869.

### CODE REVIEW

**Review by lockewritesdocs — APPROVED** [review]
LGTM 👍 Thanks @jrodewig!

File: x-pack/docs/en/security/authorization/built-in-roles.asciidoc [hunk 1]
```diff
@@ -183,8 +183,9 @@ However, this role does not grant permission to view the data in all indices.
 
 [[built-in-roles-watcher-admin]] `watcher_admin`::
 +
-Grants read access to the `.watches` index, read access to the watch history and
-the triggered watches index and allows to execute all watcher actions.
+Allows users to create and execute all watcher actions. Grants read access to
```
Comment (lcawl) [inline x-pack/docs/en/security/authorization/built-in-roles.asciidoc:186]: AFAIK this product should be capitalized:
```suggestion
Allows users to create and execute all {watcher} actions. Grants read access to
```

Comment (jrodewig) [reply x-pack/docs/en/security/authorization/built-in-roles.asciidoc:186]: Thanks for catching this @lcawl

**Comment by jrodewig** [comment]
@elasticmachine run elasticsearch-ci/eql-correctness

**Pull request merged**